js/src/launchAsync.ts

```diff
@@ -274,6 +276,21 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             src += '&omkt=' + options.uiLang;
         }
 
+        // Set option only when is true
```

Comment:
nit: Better, just trying to get you to make the comment here consistent with the comments for the other options

Reply:
got it, done